### ISSUE TITLE
play around with optimizers

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/same-different-transformers#4]
- increase learning rate for resnets
- try incorporating weight decay to see if this has any effect
